.taskcluster.yml

```diff
@@ -2,6 +2,334 @@ version: 1
+
+                          payload:
+                              env:
+                                  # run-task uses these to check out the source; the inputs
+                                  # to `mach taskgraph decision` are all on the command line.
+                                  $merge:
+                                      - MOBILE_BASE_REPOSITORY: '${baseRepoUrl}'
+                                        MOBILE_HEAD_REPOSITORY: '${repoUrl}'
+                                        MOBILE_HEAD_REF: '${head_branch}'
+                                        MOBILE_HEAD_REV: '${head_sha}'
+                                        MOBILE_HEAD_TAG: '${head_tag}'
+                                        MOBILE_REPOSITORY_TYPE: git
+                                        TASKGRAPH_BASE_REPOSITORY: https://hg.mozilla.org/ci/taskgraph
+                                        TASKGRAPH_HEAD_REPOSITORY: https://hg.mozilla.org/ci/${taskgraph.branch}
+                                        TASKGRAPH_HEAD_REV: ${taskgraph.revision}
+                                        TASKGRAPH_REPOSITORY_TYPE: hg
+                                        REPOSITORIES: {$json: {mobile: "Focus-Android", taskgraph: "Taskgraph"}}
+                                        HG_STORE_PATH: /builds/worker/checkouts/hg-store
+                                        ANDROID_SDK_ROOT: /builds/worker/android-sdk
+                                      - $if: 'tasks_for in ["github-pull-request"]'
+                                        then:
+                                            MOBILE_PULL_REQUEST_NUMBER: '${event.pull_request.number}'
+                                      - $if: 'tasks_for == "action"'
+                                        then:
+                                            ACTION_TASK_GROUP_ID: '${action.taskGroupId}'  # taskGroupId of the target task
+                                            ACTION_TASK_ID: {$json: {$eval: 'taskId'}}  # taskId of the target task (JSON-encoded)
+                                            ACTION_INPUT: {$json: {$eval: 'input'}}
+                                            ACTION_CALLBACK: '${action.cb_name}'
+                                      - $if: 'tasks_for == "github-release"'
+                                        then:
+                                            MOBILE_HEAD_TAG: '${event.release.tag_name}'
+                              features:
+                                  taskclusterProxy: true
+                                  chainOfTrust: true
+                              # Note: This task is built server side without the context or tooling that
+                              # exist in tree so we must hard code the hash
+                              image:
+                                  mozillareleases/taskgraph:decision-mobile-682fbaa1ef17e70ddfe3457da3eaf8e776c4a20fe5bfbdbeba0641fd5bceae2a@sha256:bbb2613aaab79d17e590fbd78c072d0643be40fd1237195703f84280ecc3b302
```

Comment:
We'll want to use the latest `decision-mobile` image as well:
https://hub.docker.com/layers/mozillareleases/taskgraph/decision-mobile-0e1cefb4ec46060d53bcd0eb2c0dd296210fa969998af468d693d1c6922efd1d/images/sha256-7cfb913bee636f1ab1477a09f8ad746e1f8c68fab0e2a3e1408c985d2ddc27f7?context=explore